scripts/generate_el-7-ppc64_sysroot.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+#
+# This script generates a sysroot tarball for the el-7-ppc64 (aka
+# Power 8) platform. For our builds we cross-compile most of our packages
+# on a redhat-7-x86_64 pooler system, but the sysroot must be created
+# from a native Power8 environment. Therefore, you must run this script
+# as root on a Power8 server running RHEL 7.2.
+
+sysrootdir="el-7-ppc64-sysroot"
+
+# First we remove a bunch of unneeded junk. The purpose of this is to try
+# to get rid of unneeded library files and minmize the size of the sysroot,
+# even though we'll still be pruning the libdirs in a later step.
+#
+# Note: we cannot remove libxml2-python, as it is in the dependency chain for
+# subscription-manager, which we need to keep the yum repos enabled for this
+# platform. 
+echo "Removing unneeded packages..."
+yum remove -y autofs cups-libs git-core libtiff libxslt mysql-libs opencryptoki-libs pango plymouth-core-libs python-lxml sssd-client samba-common systemtap-runtime
+
+echo "Installing critical libraries we need for building gcc and our various dependencies..."
+yum install --assumeyes rsync zlib-devel bzip2-devel libblkid-devel libcurl-devel libselinux-devel readline-devel e2fsprogs-devel xz-devel glibc-devel
+
+# Create a sysroot working area we can make further modifications to
+echo "Creating $sysrootdir and syncing sysroot directories under it..."
+rm -rf $sysrootdir
+mkdir -p $sysrootdir/usr
+rsync --archive /usr/lib64 $sysrootdir/usr/
+rsync --archive --copy-dirlinks /usr/include $sysrootdir/usr/
+
+echo "Pruning sysroot libdirs of unneeded files and directories..."
+# I always try to delete libnss* and libkrb*, but they're actually needed by libcurl, so don't do that:
+rm -r $sysrootdir/usr/lib64/{a,cracklib,d,e,f,g,k,libasound,libau,libavahi,libdev,libgio,libgnutls,liblvm,libmoz,libpython,librpm,libslang,libsoup,libsystemd,libX,libxcb,libxml,lua,m,NetworkManager,n,o,p,r,s,t,X11,x}*
+
+# Since we're relocating the sysroot, we can't have absolute symlinks
+echo "Converting absolute library symlinks to relative ones..."
+pushd $sysrootdir/usr/lib64
```

Comment:
@pdemonaco if you add `|| exit` to all pushd lines in this file we can remove the travis failures, i.e.:

```
pushd $sysrootdir/usr/lib64 || exit
```